app/src/main/java/org/simple/clinic/phone/PhoneNumberMaskerConfig.kt

```diff
@@ -6,7 +6,8 @@ import org.simple.clinic.remoteconfig.ConfigReader
 
 data class PhoneNumberMaskerConfig(
     val maskingEnabled: Boolean,
-    val proxyPhoneNumber: String
+    val proxyPhoneNumber: String,
+    val showPhoneMaskBottomSheet: Boolean = false
```

Comment:
When this is `enabled`, the secure calling message on the overdue screen must be hidden, no?

Reply:
Removed secure calling message from overdue screen